docs/source/basic_tutorials/request_parameters.md

```diff
@@ -0,0 +1,29 @@
+# Request Parameters for Text Generation Inference
+
+Text Generation Inference supports different parameters to control the generation, defining them in the `parameters`` attribute of the payload. 
+
+```bash
+curl https://j4xhm53fxl9ussm8.us-east-1.aws.endpoints.huggingface.cloud \
```

Comment:
We always give local serving as URL and we'd like to keep that for consistency I think @osanseviero